### ISSUE TITLE
Improvment suggestion

### ISSUE THREAD

**Issue by jugla** [Pouzor/freebox_player#9]
Hello,
Just to say that it works on freebox revolution.
This ticket is just to say "ok"

An improvment can be to implement a plugin in User Interface to display a remote control

Great ! Keep on the dev !
Philippe

**Comment by landaisbenj** [comment]
Hello. Yes it's work for me too on revolution.

Maybe a remote control can be add with another plug in in HACS.

I've another suggestion, why don't making this integration on media_payer entity ?
I don't have the experience, but i think we can know if player is "power on" with some open port ...
So i think we can make it on media_player, and have a media_player card on lovelace. (power on, volume, mute, maybe source with some template, etc etc...)

**Comment by landaisbenj** [comment]
D'ailleurs je ne sais pas pourquoi on parle anglais ici du coup ? :D 

En ce qui concerne la télécommande, on peut mixer peut etre avec quelque chose comme ca: 
https://github.com/marrobHD/tv-card

**Comment by jugla** [comment]
En fait, si qqn tombe sur le fil de discussion, il pourra suivre ;-)
Pour la telecommande, j'ai fait ca [https://community.home-assistant.io/t/freeplayer-freebox-player/186487/3](url https://community.home-assistant.io/t/freeplayer-freebox-player/186487)
J'ai mis toutes les touches de la télécommande !

Peut etre a mettre sous HACS ;-)

**Comment by landaisbenj** [comment]
Yes merci !! Dommage qu’on ait pas une implémentation en tant que média player. Ça aurait été cool de l’ajouter à Google home. Il faut que je trouve sur ha comment utiliser Google home Avec des phrases pré enregistrer genre ifttt.

**Comment by jugla** [comment]
Bonne idée le média player !!

Pour IFTTT sur Home Assistant, dans le principe , c'est simple.
[https://www.home-assistant.io/integrations/ifttt/](https://www.home-assistant.io/integrations/ifttt/)

Vu de IFTTT, il y a une URL avec une série d'argument (WebHook) pour appeler home assistant.
Au plus simple, programmer une série de phrase (sous IFTTT) pour Google Home avec une WebHook sur l'adresse IP de home assistant.

Amuses toi bien
Philippe

**Comment by landaisbenj** [comment]
Oui je faisais ce principe sur domoticz. Mais je pensai qu’il était plus simple de gérer ce genre de détection avec ha grâce au module qu’il existe. Notamment le sdk Google assistant. 
Je regarderai de toute façon.  J’ai encore énormément de chose à mettre en place 😕. 

Merci !
